archive/Thematic/2021/Februrary/progressed/AMA/content.md

```diff
@@ -11,8 +11,7 @@ Hey There Hackers,
 Thank you for joining our Slogging channel! We hope you’ve had a chance to share your thoughts on some active threads and perhaps even started some of your own. 
 
 
-
-{IMAGE}
+![https://gitlab.com/mikhailnathanielabordo/creative/-/raw/master/newsletters/memes/2021/Feb/13.02/tenor.gif][https://tenor.com/view/well-done-despicable-me-minions-gif-4733480][Well Done!]

```

Comment:
we cant use pathes to your repository. while you did everything well - we still need to have images under HN account. because those images can help with some SEO